code/modules/atmospherics/machinery/components/binary_devices/dp_vent_pump.dm

```diff
@@ -10,13 +10,13 @@
 	name = "dual-port air vent"
 	desc = "Has a valve and pump attached to it. There are two ports."
 
-	can_unwrench = 1
+	can_unwrench = TRUE
 
 	level = 1
 
 	connect_types = list(1,2,3) //connects to regular, supply and scrubbers pipes
 
-	var/pump_direction = 1 //0 = siphoning, 1 = releasing
+	var/pump_direction_releasing = TRUE //FALSE = siphoning, TRUE = releasing
```

Comment:
```suggestion
	var/releasing = TRUE //FALSE = siphoning, TRUE = releasing
```

Also you need to update the other map files which reference this

Reply:
Done